.github/workflows/run-match.yml

```
@@ -0,0 +1,75 @@
+name: Simulator match
+
+on:
+  push: {}
+  release:
+    types: [ created ]
+
+jobs:
+  run-comp-match:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ github.workspace }}
+    strategy:
+      matrix:
+        webots-version: ["2020b-rev1", "2021a"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          sudo apt-get update
+          sudo apt-get install --yes wget zip pulseaudio xvfb
+      - name: Download Webots
+        run: |
+          wget -O ./webots.deb \
+          https://github.com/cyberbotics/webots/releases/download/R${{ matrix.webots-version }}/webots_${{ matrix.webots-version }}_amd64.deb
+      - name: Install Webots
+        run: |
+          sudo apt-get install --yes ./webots.deb
```

Skip the `apt-get` wrapper?

```suggestion
          sudo dpkg -i ./webots.deb
```

---

Note that `dpkg` won't install dependencies, `apt-get` will do.

---

This was indeed the case